spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDefinitionController.java

```diff
@@ -16,10 +16,19 @@
 
 package org.springframework.cloud.dataflow.server.controller;
 
+import java.util.ArrayList;
```

Comment:
+1 for using Spring Framework's code conventions.  I'd be open to hearing a reason why we should deviate but in general, the order of the imports isn't something I'd expect to have to mess with as I move from Spring project to Spring project...